tests/README.md

```diff
@@ -11,5 +11,5 @@
     }
     ```
 * To run the tests contained in `test_users.py` a json with credentials for a user with the role user admin have to be provided in a file named `config_useradmin.json`. The format has to be the same as `config.json`. Both users have to live in the same organisation on the same LANA Process Mining instance.
-* We require the r analyst role to have access to one log named `Incident_management.csv`.
+* We require the r analyst role to have access to one log named `Incident_Management.csv`, one shiny dashboard named `incident-test-shiny-dashboard` and one dashboard named `incident-test-dashboard`. The dashboard and shiny dashboard don't need to be connected to a log. The shiny dashboard also doesn't need to contain actual code as we just use its metadata for the tests.
```

Comment:
Can a non-shiny dashboard not be connected to a log?

Reply:
Not through the frontend, but if a new dashboard is created via `create_dashboard()` or if an existing connection is deleted via the endpoint `DELETE /api/v2/resource/connections`, there would be a dashboard with no connection to a log.

Reply:
Thanks, I didn't know that. Then the text is very instructive.